docs/content/en/docs/tasks/cluster/cluster-scale/baremetal-scale.md

```diff
@@ -54,17 +54,34 @@ If you don't have any available hardware that match this requirement in the clus
 
 #### Upgrade Cluster Command for Scale Up/Down
 
-##### With Hardware CSV File
-
-```bash
-eksctl anywhere upgrade cluster -f cluster.yaml --hardware-csv <hardware.csv>
-```
-
-##### Without Hardware CSV File
-
-```bash
-eksctl anywhere upgrade cluster -f cluster.yaml
-```
+1. eksctl CLI: To create a workload cluster with eksctl, run:
+    ```bash
+    eksctl anywhere upgrade cluster 
+    -f cluster.yaml 
+    # --hardware-csv <hardware.csv> \ # uncomment to add more hardware
+   --kubeconfig mgmt/mgmt-eks-a-cluster.kubeconfig
+    ```
+    As noted earlier, adding the `--kubeconfig` option tells `eksctl` to use the management cluster identified by that kubeconfig file to create a different workload cluster.
+
+2. kubectl CLI: The cluster lifecycle feature lets you use kubectl to talk to the Kubernetes API to upgrade a workload cluster. To use kubectl, run:
```

Comment:
Users can also add hardware before trying to upgrade clusters, we can add the steps here too for using eksctl anywhere generate hardware.

Reply:
It's already mentioned [here](https://github.com/aws/eks-anywhere/pull/5388/files#diff-b516ccc1914bec43097746019578989d8d81a60724e389e1fa844afa3a665aaeR61)